contracts/examples/ping-pong-egld/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
```

Comment:
Last week we added ABI functionality.
ABIs are basically some json files describing the public contract functions.
Anyway, to enable for this contract too just copy the "abi" sub-crate from another contract and change the names (after merge from master). It's similar to how we have a "wasm" sub-crat to every contract.

Also make sure you add your contract to build-wasm.sh.

Reply:
done